src/Polly.Specs/Helpers/Bulkhead/AssertionFailure.cs

```diff
@@ -4,7 +4,7 @@ public class AssertionFailure
 {
     public AssertionFailure(int expected, int actual, string measure)
     {
-        if (string.IsNullOrWhiteSpace(measure)) throw new ArgumentNullException(nameof(measure));
+        Guard.NotNull(measure);
```

Comment:
Did you mean to change this to allow white space? I think it's the only occurrence in the diff, and it's in a test, but thought I'd ask to check.